static/src/js/components/GranuleResults/GranuleDownloadButton.jsx

```diff
@@ -104,37 +108,56 @@ export const GranuleDownloadButton = (props) => {
     }
   }
 
+  const stringifiedProjectParams = stringify({
+    ...params,
+    p,
+    pg
+  })
+
+  const downloadButtonProps = {
```

Comment:
This might be a personal thing. I am not sure theres a hard line in the sand. But I feel like defining props as an object like this hurts readability. 

We generally have only done this sort of thing only when: a) we have conditional props that need to be omitted in certain cases, or b) when we want to forward an unknown props to a element.

Reply:
I suggested this because we had so many props that were exactly the same between the two buttons. I was just trying to remove some of that duplication

Reply:
or c) when we want to share the same props across conditionally rendered components...

:ill_see_myself_out:

Reply:
The deduplication here was part of Mathew's feedback when pairing.  I am ambivalent about its use here as I don't experience the readability issue that you do.  It does make it more concise so if I had to choose I'd pick the way it currently is.